crates/trippy/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "trippy"
+description = "A network diagnostic tool"
+version.workspace = true
+authors.workspace = true
+documentation.workspace = true
+homepage.workspace = true
+repository.workspace = true
+readme.workspace = true
+license.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+keywords = ["cli", "tui", "traceroute", "ping", "icmp"]
+categories = ["command-line-utilities", "network-programming"]
+
+[[bin]]
+bench = false
+path = "src/main.rs"
+name = "trip"
+
+[dependencies]
+trippy-core = { version = "0.11.0-dev", path = "../trippy-core" }
+trippy-privilege = { version = "0.11.0-dev", path = "../trippy-privilege" }
+trippy-dns = { version = "0.11.0-dev", path = "../trippy-dns" }
+thiserror.workspace = true
+anyhow.workspace = true
+itertools.workspace = true
+tracing.workspace = true
+serde.workspace = true
+derive_more.workspace = true
+parking_lot.workspace = true
+clap = { version = "4.4.0",  default-features = false, features = [ "cargo", "derive", "wrap_help", "usage", "unstable-styles" ] }
+clap_complete = "4.4.9"
+humantime = "2.1.0"
+ratatui = "0.26.2"
+crossterm = { version = "0.27.0",  default-features = false, features = [ "events", "windows" ] }
+chrono = { version = "0.4.38", default-features = false, features = [ "clock" ] }
+serde_json = { version = "1.0.117", default-features = false }
+comfy-table = { version = "7.1.0", default-features = false }
+strum = { version = "0.26.2", default-features = false, features = [ "std", "derive" ] }
+etcetera = "0.8.0"
+toml = { version = "0.8.13", default-features = false, features = [ "parse" ] }
+indexmap = { version = "2.2.6", default-features = false }
+maxminddb = "0.24.0"
+tracing-subscriber = { version = "0.3.18", default-features = false, features = [ "json", "env-filter" ] }
+tracing-chrome = "0.7.2"
+petgraph = "0.6.5"
+csv = "1.3.0"
+serde_with = "3.8.1"
+encoding_rs_io = "0.1.7"
+clap_mangen = "0.2.20"
+
+[dev-dependencies]
+test-case.workspace = true
+serde_yaml.workspace = true
+pretty_assertions = "1.4.0"
+
+# cargo-generate-rpm dependencies
+[package.metadata.generate-rpm]
+assets = [
+  { source = "target/release/trip", dest = "/usr/bin/trip", mode = "755" },
+]
```

Comment:
I'll have to test it by doing a release (from my testing shadow repo).  I suspect this can be replaced with a few command line arguments added to the GitHub action step.

Reply:
Fixed by adding `-p crates/trippy` to the `generate-rpm` command.  

Aside: `cargo-generate-rpm` [supports](https://github.com/cat-in-136/cargo-generate-rpm?tab=readme-ov-file#overwrite-configuration) inline toml configuration (via the `-s` flag) but I couldn't get it to work for the needed which has an array of nested objects:

```toml
assets = [
  { source = "target/release/trip", dest = "/usr/bin/trip", mode = "755" },
]
```